### PR TITLE
Fixes and improvements

### PR DESCRIPTION
**Pull request by Grabstertv**
- Added section-background CSS variable to common.css for when more content is eventually added to landing page.
- Changed section background color so it doesn't look like a button.
- Fixed margin issues in downloads page by introducing new IDs to some HTML elements.
- Removed scrollbar buttons from taking empty space by setting it to display: none
- Reduced scrollbar track's border radius to 0 so it doesn't have black corners on top and bottom.
- Increase logo brightness in dark mode.
- Increased content width in landing page.
- Aligned hero content to a grid so its in line with the rest of the page content.
- Feature text now uses 100% width of its container and is also in line with page content.
- Hero background image now uses 100% viewport width.

### CODE REVIEW

File: assets/common.css [hunk 1]
```diff
@@ -52,7 +55,6 @@
 
 * {
   box-sizing: border-box;
-  transition: outline 0s;
```
Comment (SuperDragonXD) [inline assets/common.css]: This is used to prevent the weird animations when navigating using the keyboard

File: assets/faq.css [hunk 2]
```diff
@@ -183,6 +183,13 @@ code {
 }
 
 @media screen and (max-width: 700px) {
+
+  main div:first-child {
```
Comment (SuperDragonXD) [inline assets/faq.css]:
```suggestion
  main div:first-child {
```

File: assets/index.css [hunk 3]
```diff
@@ -234,25 +239,23 @@ main>section:not(:first-child) {
...
-}
-
 @media screen and (prefers-color-scheme: dark) {
   .hero__content {
     color: var(--text-color);
   }
 
   .hero__content__icon {
-    filter: brightness(75%);
+    filter: brightness(90%);
```
Comment (SuperDragonXD) [inline assets/index.css:250]: Just to check, will it be blinding to the user or not?

File: assets/index.css [hunk 4]
```diff
@@ -234,25 +239,23 @@ main>section:not(:first-child) {
...
   }
 
   .hero__content__description {
     margin-bottom: 30px;
   }
 
+  .about__info {
+    width: 100%;
+  }
+
```
Comment (SuperDragonXD) [inline assets/index.css]: Why should the about info be 100% on dark mode?

Also I may have misplaced the
```css
  .hero__content__description {
    margin-bottom: 30px;
  }
```

text. It should be in the `@media max-width: 800` (or 520?) part.

File: assets/downloads.css [hunk 5]
```diff
@@ -32,15 +32,18 @@ img {
   margin-bottom: 10px;
 }
 
-.card__right {
+#lawnchair_title {
+  padding-top: 10px;
+}
+
+#card_main_cta {
```
Comment (SuperDragonXD) [inline assets/downloads.css]: Can't you simply use `.card__main h2` and `.card__main .card__right`?

File: downloads/index.html [hunk 6]
```diff
@@ -52,10 +52,10 @@ <h1 style="font-size: 0.0em">Lawnchair Downloads</h1>
           <source type="image/webp" srcset="/images/lawnchair.webp" >
           <img src="/images/lawnchair.png" width="96" height="96" alt="The Lawnchair Logo" >
         </picture>
-        <h2 class="card__name" title="The actual launcher.">Lawnchair</h2>
+        <h2 class="card__name" id="lawnchair_title" title="The actual launcher.">Lawnchair</h2>
         <span class="card__version" id="js-lawnchair-version">Loading...</span>
       </div>
-      <div class="card__right">
+      <div class="card__right" id="card_main_cta">
```
Comment (SuperDragonXD) [inline downloads/index.html]: You can remove the class names and use selectors instead (See my comment in `assets/downloads.css`).

File: index.html [hunk 7]
```diff
@@ -64,6 +65,7 @@
       <a href="/faq">FAQ</a>
     </nav>
   </header>
+
```
Comment (SuperDragonXD) [inline index.html:68]: Purpose of the extra whitespace? It's already readable (and you can collapse the tags anyways)

File: assets/index.css [hunk 8]
```diff
@@ -19,17 +19,19 @@ main>*:not(:first-child) {
   background-color: var(--lawnchair-background-color);
   background-repeat: no-repeat;
   background-size: cover;
-  min-height: 90vh;
+  min-height: 100vh;
```
Comment (SuperDragonXD) [inline assets/index.css]: The reason this isn't full width is to show that users can scroll the page. (Sometimes, scrollbars aren't enough to show the page is scrollable. Also mobile doesn't show them)

File: assets/index.css [hunk 9]
```diff
@@ -294,10 +297,16 @@ main>section:not(:first-child) {
...
+  .links__cards {
+    flex-wrap: nowrap;
+    flex-direction: column;
+  }
+
   .link__card {
     min-width: -webkit-fill-available;
     min-width: fill-available;
     padding: 16px 20px;
+    max-width: inherit;
```
Comment (SuperDragonXD) [inline assets/index.css:312]: Have done some tweaks to them (for a11y), so should be unnecessary.

**Review by SuperDragonXD — CHANGES_REQUESTED** [review]
Thanks for creating this PR! Have a few changes in mind.

Also, what's the purpose of the videos if you haven't added the feature yet? I'd recommend maybe putting it later (like in a different branch) until you're sure that you have fully made that.

(Also, I'd recommend requesting me for review and assigning me to this PR, so I won't ignore it.)

File: assets/common.css (hunk 1 above)
Comment (Grabstertv) [reply assets/common.css]: Transition does not play in this case so there is no point keeping it here. The border is still visible when navigating on keyboard.
image

File: assets/index.css (hunk 3 above)
Comment (Grabstertv) [reply assets/index.css:250]: Yes. Has a good contrast ratio without blinding users. Logo colors were being affected at 75%.

File: assets/faq.css (hunk 2 above)
Comment (Grabstertv) [reply assets/faq.css]: I don't see any change here?

File: assets/downloads.css (hunk 5 above)
Comment (Grabstertv) [reply assets/downloads.css]: Correct. Will replace in next commit.

File: assets/faq.css (hunk 2 above)
Comment (SuperDragonXD) [reply assets/faq.css]: Oh, it's just a formating change (remove the line break).

**Review by SuperDragonXD — APPROVED** [review]

**Pull request merged**